post_results.py

```diff
@@ -19,11 +21,20 @@
 requesttimeout = 5
 processingtimeout = 300
 
+slacktoken = ""
```

Comment:
why not do this all inside the slack function, it is not used anywhere else.

Reply:
done